OpenRA.Mods.Cnc/Traits/SupportPowers/ChronoshiftPower.cs

```diff
@@ -241,9 +244,9 @@ public IEnumerable<IRenderable> RenderAboveShroud(WorldRenderer wr, World world)
 				var palette = wr.Palette(power.Info.IconPalette);
 
 				// Destination tiles
-				foreach (var t in world.Map.FindTilesInCircle(xy, range))
+				foreach (var t in world.Map.FindTilesInCircle(xy, range, true))
 				{
-					var tile = manager.Self.Owner.Shroud.IsExplored(t) ? validTile : invalidTile;
+					var tile = manager.Self.Owner.Shroud.IsExplored(t) && world.Map.Contains(t) ? validTile : invalidTile;
```

Comment:
Ok, this is weird. Now that I look at it, `IsExplored` already has a `Map.Cotains` check: https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Traits/Player/Shroud.cs#L331
So this shouldn't be necessary, or something else is going on... (Maybe this [here](https://github.com/OpenRA/OpenRA/blob/bleed/OpenRA.Game/Traits/Player/Shroud.cs#L341)?)